### PR TITLE
admin activity log: instructor home page ajax requests are shown as null #4719

### LINKED ISSUE #4719: admin activity log: instructor home page ajax requests are shown as null
image

### PR DESCRIPTION
**Pull request by belyabl9**
Fixes #4719

### CODE REVIEW

File: src/main/java/teammates/ui/controller/FeedbackSessionStatsPageAction.java [hunk 1]
```diff
@@ -29,6 +29,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 false);    
         
         data.sessionDetails = logic.getFeedbackSessionDetails(feedbackSessionName, courseId);
+
+        statusToAdmin = "Load feedback session details:<br>" + "Session name: " + feedbackSessionName;
```
Comment (bamboo3250) [inline src/main/java/teammates/ui/controller/FeedbackSessionStatsPageAction.java]: This issue is only about InstructorHomePage so I think there is no need to do for other places here. Instead, you can open a new issue to to add `statusToAdmin` to other Actions that don't have it.

**Comment by bamboo3250** [comment]
We need to modify `statusToAdmin` before we return in line 26 as well.

**Comment by belyabl9** [comment]
Ready to review

**Comment by thyageshm** [comment]
@bamboo3250 see if you can merge in the next hour. if not, no rush

**Comment by bamboo3250** [comment]
@thyageshm does it have to be before midnight? Since I'm going to merge all PRs tonight.

**Pull request merged**